frontend/src/metabase/admin/settings/components/SettingsLdapForm.unit.spec.tsx

```diff
@@ -16,225 +20,23 @@ const GROUPS = [
   createMockGroup({ id: 5, name: "flamingos" }),
 ];
 
-const elements = [
-  {
```

Comment:
since we just define the form keys in the form now ... we don't need any of this to test it.